corehq/apps/users/migrations/0024_add_login_as_all_users.py

```diff
@@ -23,8 +9,6 @@ class Migration(migrations.Migration):
         ('users', '0023_hqapikey_role_id'),
     ]
 
-    operations = [
-        migrations.RunPython(migrate_login_as_all_users_permission,
-                             reverse_code=migrations.RunPython.noop,
-                             elidable=True)
-    ]
+    operations = {
+        block_upgrade_for_removed_migration('4c7d3a96e061680bc87567d5916ae1e90dd60858')
```

Comment:
I did debate which strategy to use, the older commit is going to be closer to their current state but the newer commit get's them further along. I guess it probably makes most sense to use a newer commit since if there are any other migrations that also block then they would find out sooner instead of only when the re-run the deploy.

Reply:
7f9ad1eb670